_capabilities/utdallas-ufl-xar-demo.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Explainable Video Activity Recognition Demo"
+excerpt: "Demo of our XAI system built to detect activities in videos with post-hoc explanations for its predictions."
+tags: 
+- Analytics
+- Computer Vision
+- Visual Question Answering (VQA)
+- Human-Machine Teaming
+- Explanation Framework
+
+submission_details:
+resources: 
+papers: 
+- title: Don’t Explain without Verifying Veracity: An Evaluation of Explainable AI with Video Activity Recognition
+url: https://arxiv.org/abs/2005.02335
+software:
+- title: Explainable Video Activity Recognition Demo
+url: https://github.com/MahsanNourani/XAI-Video-Explanation
+
+
```

Comment:
Please also leave the version, size, and license tags empty if unused (this helps with proper rendering on the website):

```suggestion
  version:
  size:
  license:
```